keymanapp/keyman
# chore(common): Check in crowdin strings for Fulfulde Nigeria

These use the locale `ff-NG` in contrast to the existing Pulaar-Fulfulde locale `ff-ZA`.

## User Testing
For all the platforms, load the PR build, set the UI to "Ɗemngal Fulfulde" (sometimes listed as Fulfulde Nigeria), and verify the UI is in Fulfulde

* **TEST_ANDROID** - Tests Ɗemngal Fulfulde UI for Android
1. Load the PR build in an Android 5.0 emulator (SDK 21)
2. In the Keyman app, go to Settings --> Display Language --> Ɗemngal Fulfulde
3. Verify the UI is in Ɗemngal Fulfulde

* **TEST_IOS** - Tests Ɗemngal Fulfulde UI for iOS
1. Start the app
2. Under Settings - General - Language & Region, set the Language to Pulaar Fula and the Region to Nigeria
3. Verify the app refreshes with the Fulah Nigeria, or 'ff-NG' strings

* **TEST_WINDOWS** - Tests Ɗemngal Fulfulde UI for Windows

* **TEST_LINUX** - Tests Ɗemngal Fulfulde UI for Linux
1.  start Keyman configuration with: `LANGUAGE=ff_NG km-config`
